zappa/__init__.py

```diff
@@ -1,16 +1,16 @@
+import os
 import sys
-from pathlib import Path
 
 
+# Alternative solution to: https://github.com/zappa/Zappa/issues/1140
```

Comment:
This comment is unnecessary, can you remove it?

Reply:
Done